src/client_side.h

```diff
@@ -383,6 +387,10 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
     /// whether PROXY protocol header is still expected
     bool needProxyProtocolHeader_;
 
+    /// the parsed PROXY protocol message or nil if there is
+    /// no PROXY protocol configuration for the HTTP port
+    ProxyProtocol::Message::Pointer theProxyProtocolMessage;
```

Comment:
s/theProxyProtocolMessage/proxyProtocolMessage_/ to follow modern convention for naming private data members.